services/drupal/web/modules/custom/view_mode_select_by_token/config/schema/view_mode_select_by_token.schema.yml

```diff
@@ -0,0 +1,7 @@
+field.formatter.settings.epa_core_rendered_with_select:
+  type: mapping
+  label: 'Rendered With Select'
+  mapping:
+    text_length:
+      type: string
+      label: 'View Mode'
```

Comment:
I applaud you for including the schema file but I'd have expected the key for this to be `field.formatter.settings.view_mode_select_by_token_rendered_with_select` and the mapping to reference `view_mode` (rather than `text_length`) along with an entry for `link` since we're still preserving that setting.